spec/acceptance/foreman_basic_spec.rb

```diff
@@ -16,4 +16,24 @@
   describe package('foreman-telemetry') do
     it { is_expected.not_to be_installed }
   end
+
+  context 'GSSAPI auth enabled' do
+    let(:apache_service) { %w[debian ubuntu].include?(os[:family]) ? 'apache2' : 'httpd' }
```

Comment:
Technically it needs to be a symbol:
```suggestion
    let(:apache_service) { %w[debian ubuntu].include?(os[:family]) ? 'apache2' : 'httpd' }
```

But this is part of the [shared example](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples), defined in https://github.com/theforeman/puppet-foreman/blob/master/spec/support/acceptance/examples.rb. So it'd be sufficient to just use the `it_behaves_like` part.

Reply:
Sorry, a typo I've noticed by failed tests as well :/

I've removed duplicate tests for service, thanks for pointing out :)